llvm/lib/Transforms/Utils/CloneFunction.cpp

```diff
@@ -200,6 +200,22 @@ bool llvm::BuildDebugInfoMDMap(DenseMap<const Metadata *, TrackingMDRef> &MD,
   return ModuleLevelChanges;
 }
 
+void llvm::CloneFunctionMetadataInto(Function *NewFunc, const Function *OldFunc,
+                                     ValueToValueMapTy &VMap,
+                                     RemapFlags RemapFlag,
+                                     ValueMapTypeRemapper *TypeMapper,
+                                     ValueMaterializer *Materializer) {
+  // Duplicate the metadata that is attached to the cloned function.
```

Comment:
Note that, in the context of this function, there is no "cloned function", it's either `NewFunc` or `OldFunc`. 

That  said, I this comment fits better in the documentation of the header, as there is important information here that callers should be aware

Reply:
Thanks! Moved relevant documentation bits into Cloning.h.